ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -13,10 +13,10 @@ struct ToDoSection: Identifiable {
     let toDoItems: [TaskModel]
 }
 struct ToDoListView: View {
-    @StateObject var viewModel = ToDoListViewModel()
+    @StateObject var viewModel: ToDoListViewModel = ToDoListViewModel()
     var body: some View {
-        NavigationStack{
-            Group {
+        NavigationStack {
+            VStack(alignment: .trailing) {
```

Comment:
If the add button will be inside a VStack, then on the UI it will be placed below the list on Y axis, but it should be above the list on Z axis according to Figma. In order to stack views on top of each other you need to use ZStack